changelog.mdx

```diff
@@ -5,6 +5,43 @@ metaDescription: Chalk's latest updates.
 description: Updates to Chalk!
 ---
 
+## December 23, 2024
+
+### Search and filter logs by query name, log message, deployment, and more!
+
+You can now search logs in the Logs Explorer in the dashboard using a key-value pair query format:
+`key1: value1 key2: value2 ...`. This allows you to combine multiple filters at once. The Logs Explorer
```

Comment:
Is there a space between the key value pairs or is it `key1:value1`

Reply:
This is probably beyond the scope of this change log since we've called in `Logs Explorer` in the dashboard, but datadog calls theirs `Log Explorer`. I wonder if we should do the same.
Like i recognize we are like providing access to multiple different logs but...yah... idk haha

Reply:
good catch